### DISCUSSION TITLE
how to install Data Loader on Linux?

### DISCUSSION THREAD

**Discussion by ashitsalesforce** [forcedotcom/dataloader#847] [Q&A] [answered]
From @souvsark:
First of all i would like to be honest - i am not a software guy and i do not have technical background. I am a business analyst and working in automobile domain. Recently i was going through Salesforce admin course and there i came across a concept of dataloader. Now i have a linux OS in my personal laptop and i am struggling to install dataloader in it. I have watched videos but somewhere i am missing out something i have Linuxmint installed in the machine. can anybody please help ?

**Comment by ashitsalesforce** [comment, marked as answer]
Hi @souvsark,

Data Loader is not supported on Linux. However, many users have successfully installed it on linux and have been able to use it. Have you downloaded the latest version, [59.0.0](https://github.com/forcedotcom/dataloader/releases/tag/v59.0.0)? Once you download it, extract from the zip file, rename "install.command" as "install.sh", and double-click on install.sh to start the installation. Answer the prompts to install data loader. If double-click does not work, you may have to launch a command terminal, navigate to the folder containing install.sh, and run it by entering `./install.sh`. Once installed, you will see `dataloader.sh` in the installation folder. You can launch it from the command line or by double-clicking.

**Reply by souvsark** [reply]
Hi Asit,

Thanks a lot. Your solution worked absolutely fine. i had to install openjdk version 7 first on the machine and then followed the steps you mentioned. i could run the dataloader application from linux. 

Thanks again

**Reply by ashitsalesforce** [reply]
Hi @souvsark, thanks for letting us know. Good to hear that it is working for you.
